docs/content/en/docs/concepts/executable-playbooks.md

```diff
@@ -0,0 +1,387 @@
+
+The `condition` field contains a string that specifies a [STIX Pattern](https://docs.oasis-open.org/cti/stix/v2.0/stix-v2.0-part5-stix-patterning.html). Currently, SOARCA only supports a very small subsection of the STIX Patterning specification. We support string based equality (`'a' = 'a'`) and inequality (`'a' != 'b'`) comparison. Example:
+
+```json
+"if-condition--4b95eaa4-944a-4a9d-88d4-1374a70dbacd": {
+    "name": "If it is not new years",
+    "description": "Checks if it is 01-01-2025",
+    "on_completion": "end--db937fc8-3a42-41cc-b828-ec2db212f425",
+    "type": "if-condition",
+    "condition": "__soarca_ssh_result__:value != '01-01-2025'",
+    "on_true": "action--7fe08053-3685-4d8c-bc0a-40efce75113e"
+}
+```
+
+SOARCA supports variable interpolation, which means that variables can be used inside the `condition` field, as seen in the example above.
+
+Similarly, CACAO specifies `while-condition` steps, whose `on_true` branch will be repeatedly executed until the condition evaluates to false. The execution limit is currently set to 10. If the condition is still true after 10 iterations, the step ends in a failure.
+
+The `parallel` step allows executing multiple branches (in parallel) specified in the `next_steps` field. At this time, the steps in `next_steps` are executed sequentially. Parallel execution is scheduled for a later release in SOARCA.
+
+Next, we explain variables in CACAO and SOARCA.
+
+### Variables
+
+The CACAO specification allows defining variables on the playbook level, as well as on the step level. Playbook variables are available throughout the playbook. Variables defined on the step level are available in that step, and in any step that executes in a sub-branch of an `if-condition`, `while-condition`, or `parallel` step.
+
+According to the CACAO spec, variable names should start and end with double underscores (`__`), but this is not enforced by SOARCA. Furthermore, the CACAO spec allows defining multiple types of variables (strings, ip-addresses, numbers), but at this time SOARCA will interpret every value as a string. The `constant` and `external` fields are also ignored.
+
+```json
+"playbook_variables": {
+    "__target_ip__": {
+      "type": "ipv4-addr",
+      "description": "IP address of target system",
+      "constant": false,
+      "external": true
+    }
+}
+```
+
+SOARCA supports the interpolation of variables in different strings. The specific string-based fields that support interpolation depend on the capability. In general, string interpolation is supported in the fields of agents, targets, and `command` fields.
```

Comment:
We use "field" often in the docs instead of "property", which is the term the specification uses. I would consider changing all "field" mentions to "property"

Reply:
Done